src/main/java/com/google/sps/data/DatastoreScheduledInterviewDao.java

```diff
@@ -133,4 +155,16 @@ public Entity scheduledInterviewToEntityForUpdate(ScheduledInterview scheduledIn
     scheduledInterviewEntity.setProperty("interviewee", scheduledInterview.intervieweeEmail());
     return scheduledInterviewEntity;
   }
+
+  private List<Entity> getEntitiesInRange(long minTime, long maxTime, Optional<Filter> filterOpt) {
```

Comment:
Lots of duplication from the same exact function in availability Dao, we can ask about this at the standup.
l don't know if it would be worth/safe to abstract this out into another class or something.